addon/index.d.ts

```diff
@@ -739,6 +739,32 @@ export function allSettled<T extends readonly unknown[] | readonly [unknown]>(
 ): Promise<{ -readonly [K in keyof T]: Settled<T[K]> }>;
 export function allSettled<T>(values: Iterable<T>): Promise<Array<Settled<T>>>;
 
+/**
+ * Yielding `animationFrame()` will pause a task until after the next animation
+ * frame using the native `requestAnimationFrame()` browser API.
+ *
+ * The task below, when performed, will print the time since the last loop run
+ * for every animation frame.
+ *
+ * ```js
+ * export default Component.extend({
+ *   myTask: task(function * () {
+ *     let lastNow = performance.now();
+ *     while (true) {
+ *       yield animationFrame(1000);
```

Comment:
we don't need this argument, right?

Reply:
whoops, you're totally right. copy-paste mistake 🤦